keyboards/crkbd/keymaps/frogpad/keymap.c

```diff
@@ -0,0 +1,176 @@
+    ),
+
+    [_EXT] = LAYOUT( \
+/*I recommend adding a TO() that leads to your two-handed layer to the first and last key in the next line of code if you 
+are adding this keymap to a two-handed keymap. It is left blank for this purpose.*/
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+      XXXXXXX, KC_WBAK, KC_HOME,   KC_UP,  KC_END, KC_WFWD,                      KC_WBAK, KC_HOME,   KC_UP,  KC_END, KC_WFWD, XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX,    ATAB, KC_LEFT, KC_DOWN, KC_RGHT,  KC_DEL,                       KC_DEL, KC_LEFT, KC_DOWN, KC_RGHT,    ATAB, XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX, C(KC_Z), C(KC_X), C(KC_C), C(KC_V),    OALT,                         OALT, C(KC_V), C(KC_C), C(KC_X), C(KC_Z), XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                             OSFT,    OCTL,    CBSP,       CBSP,    OCTL,    OSFT \
+                                      //`--------------------------'  `--------------------------'
+    )
+  
+/* You should add these layers on to your existing keymap so you can toggle between one-handed usage and two-handed usage. Don't 
+forget to change the first keycode in _EXT to whatever you name your default layer aside from the frogpad layers. If you add 
+a TO(_FROG) somewhere on your default layer, you can easily switch between the two.
+  
+Make sure to add "#include "g/keymap_combo.h"" to the top of your keymap.c and add combos.def to the keyboard folder. 
+Also add "VPATH  +=  keyboards/gboards/" and "COMBO_ENABLE = yes" to rules.mk."
+  
+If you're having trouble with the OSM modifiers include the code for individual tapping terms in this keymap. 
+You must also use the last six 'define's in the config.h file. */
+
+};
+
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+// Setting ADJUST layer RGB back to default
+void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
+  if (IS_LAYER_ON(layer1) && IS_LAYER_ON(layer2)) {
+    layer_on(layer3);
+  } else {
+    layer_off(layer3);
+  }
+}
+
+void matrix_init_user(void) {
+    #ifdef RGBLIGHT_ENABLE
+      RGB_current_mode = rgblight_config.mode;
+    #endif
+    //SSD1306 OLED init, make sure to add #define SSD1306OLED in config.h
+    #ifdef SSD1306OLED
+        iota_gfx_init(!has_usb());   // turns on the display
+    #endif
+}
+
+//SSD1306 OLED update loop, make sure to add #define SSD1306OLED in config.h
+#ifdef SSD1306OLED
```

Comment:
This should be replaced with the equivalent OLED Driver code.  

You can see examples of this in the oled_sample keymap.